__tests__/app.test.js

```diff
@@ -154,4 +155,47 @@ describe("API Articles", () => {
       })
     })
   })
-});
+  describe("GET /api/articles/:article_id/comments", () => {
+    it("200 return the comments for an article_id, with comment_id, votes, created_at, author, body, article_id. Sorted most recent first", () => {
+      return request(app)
+      .get('/api/articles/1/comments')
+      .expect(200)
+      .then(({body}) => {
+        const firstComment = body.comments[0]
```

Comment:
Please check the length of the array here as well. An if statement is not enough.